python_modules/dagster-graphql/dagster_graphql/schema/resources.py

```diff
@@ -86,6 +94,7 @@ class GrapheneResourceDetails(graphene.ObjectType):
     )
     resourceType = graphene.NonNull(graphene.String)
     assetKeysUsing = graphene.Field(non_null_list(GrapheneAssetKey))
+    jobsOpsUsing = graphene.Field(non_null_list(GrapheneJobWithOps))
```

Comment:
Any reason we don't just re-use the existing `GrapheneJob` objects here? Just kinda leaves a weird taste in my mouth to have this super bespoke gql schema when we have something else that already has all this info

Reply:
Updated, now returns `GrapheneJob` and `GrapheneSolidHandle`s